Localization.fr.json

```diff
@@ -0,0 +1,116 @@
+{
+    "Font": "Regular",
+    "TranslationBy": "@edgar.ogh@github.com",
```

Comment:
As the README explains, this text needs to read "Translation by \<name\>", not just a name.